linux/keyman-config/keyman_config/kvk2ldml.py

```diff
@@ -349,7 +349,11 @@ def convert_ldml(kvkData):
             else:
                 keymaps["shift"] = (uskey,)
 
-    ldml = etree.Element("keyboard", locale="zzz-keyman")
+    if kvkData.UnicodeFont:
+        font = kvkData.UnicodeFont.name
+    else:
+        font = kvkData.AnsiFont.name
```

Comment:
We should be using the font data from kmp.json, not from the .kvk file. See also #9955 for same issue on Windows.

Also, .AnsiFont is very much deprecated :)

Reply:
(One possibility that I have just considered is updating the compiler so that it rewrites the .kvk when bundling in the .kmp, but I am not sure that that is the most appropriate way forward. The OSKFont property in the .kmp is the documented font for use.